webapp/channels/src/components/global_header/left_controls/product_menu/product_branding_team_edition/product_branding_team_edition.test.tsx

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {shallow} from 'enzyme';
+import React from 'react';
+
+import ProductBrandingTeamEdition from './product_branding_team_edition';
+
+describe('components/ProductBrandingTeamEdition', () => {
+    test('should show correct product branding for team edition', () => {
```

Comment:
I don't think this is a useful test. I think we are trying to move away from these snapshot tests.

Reply:
It is just replicating the already existing `product_branding` tests.

Reply:
I would suggest just removing it. I think it's worse then useless.